src/pages/products/sgid/recreation/avalanche-center-forecast-zones.astro

```diff
@@ -0,0 +1,87 @@
+---
+import Layout from '@layouts/DataPage.astro';
+
+import type { IMetadata, IPageMetadata } from '@models/types';
+import { ProductType, SgidCategory } from '@models/types';
+
+import HubDownloads from '@components/data/HubDownloads.astro';
+import Metadata from '@components/data/Metadata.astro';
+import UpdateHistory from '@components/data/UpdateHistory.astro';
+
+import CardWithPopularLink from '@components/page/CardWithPopularLink.astro';
+import CardWithSmallLink from '@components/page/CardWithSmallLink.astro';
+import GridForMoreResources from '@components/page/GridForMoreResources.astro';
+import GridForYouMightLike from '@components/page/GridForYouMightLike.astro';
+import Section from '@components/page/Section.astro';
+
+import ExternalLink from '@components/react/ExternalLink';
+import { dataPages } from '@data/downloadMetadata';
+
+export const metadata: IMetadata = {
```

Comment:
i would suggest adding the avy website as the application here and replacing the one down in the you might also like with the know before you go program or something.